app/src/app/components/view-profile/view-profile.component.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright © 2005 - 2021 Alfresco Software, Ltd. All rights reserved.
+ *
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+import { AlfrescoApiService } from '@alfresco/adf-core';
+import { PeopleApi, Person } from '@alfresco/js-api';
+import { Component, OnDestroy, OnInit, ViewEncapsulation } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { throwError } from 'rxjs';
+@Component({
+    selector: 'adf-view-profile',
+    templateUrl: './view-profile.component.html',
+    styleUrls: ['./view-profile.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+export class ViewProfileComponent implements OnInit, OnDestroy{
+  peopleApi: PeopleApi;
+
+  profileForm: FormGroup;
+  person_details: Person;
+
+  general_section_dropdown:boolean=true;
```

Comment:
incorrect formatting of the code in this file

Reply:
changed.